### PR TITLE
add docs for new contributors

### PR DESCRIPTION
**Pull request by Rupeshiya**
---
name: Pull Request template
about: Describe the PR.
title: ''
labels: ''
assignees: ''

---

### **Problem**
Github Issue Number: #179 
Improve docs for new contributors

### **Solution of problem**
Add the links of the docs for reference and describe the workflow to follow to use the platform

### **Type of Change**
[ ] Bug fix
[ ] New Feature
[ ] Development of UI/UX prototypes
[ ] Small refactor
[&check;] Change in Documentation

### **Checklist**
[&check;] My code follows the same style as the codebase
[&check;] My Code change requires a change in documentation
[&check;] I have updated the Readme accordingly
[&check;] I made PR against **development branch**
[&check;] I have run the test cases locally and it's passing.
[&check;] I have squashed my commits

### CODE REVIEW

**Comment by Rupeshiya** [comment]
@vaibhavdaren Plz review

**Review by ksraj123 — CHANGES_REQUESTED** [review]
Some humble suggestions that may help in improving the experience of new contributors.

Please see my comments.

File: README.md [hunk 1]
```diff
@@ -122,5 +117,21 @@ DELETE  : To delete resource
         <td><code>409</code></td>
         <td><code>Conflict</code></td>
         <td>resourse with given id already exist.</td>
+    </tr>  
+    <tr>
+        <td><code>429</code></td>
+        <td><code>Too many requests</code></td>
```
[the remark this answers is not in the document]
Comment (Rupeshiya) [reply README.md:123]: @ksraj123 Please check this https://github.com/codeuino/social-platform-donut-backend/pull/178

File: README.md [hunk 2]
```diff
@@ -21,47 +21,42 @@
     </a>
 </p>
 
-### STEPS
+## Install and setup Locally
 
-```.sh
-npm install
-```
+Please follow the steps provided in the documentation to setup locally: [Documentation](https://docs.codeuino.org/donut-social-networking-platform/installations/pre-requisite)
```
[the remark this answers is not in the document]
Comment (Rupeshiya) [reply README.md:26]: @jaskiratsingh2000 Should I write that here too?

File: README.md [hunk 3]
```diff
@@ -21,47 +21,42 @@
...
-#### Package descriptions
 
-- bcrypt :- hash your plain password and store hashed password in database.
-- body-parser :- Parse incoming request bodies in a middleware before your handlers, available under the req.bodyproperty.
-- express :- Express is a minimal and flexible Node.js web application framework that provides a robust set of features for web and mobile applications.
-- jsonwebtoken :- JSON Web Token (JWT) is an open standard (RFC 7519) that defines a compact and self-contained way for securely transmitting information between parties as a JSON object.
-- mongoose :- Mongoose is a MongoDB object modeling tool designed to work in an asynchronous environment.
-- morgan :- HTTP request logger middleware for node.js.
-- nodemon :- nodemon will watch the files in the directory in which nodemon was started, and if any files change, nodemon will automatically restart your node application.
+NOTE: Make sure you have installed the pre-requisite and setup the environment variables in **.env.dev** (for development) and **.env.test** (for unit test) provided in the documentation.
```
[the remark this answers is not in the document]
Comment (Rupeshiya) [reply README.md:29]: @jaskiratsingh2000 Should I write that here too in Readme?

**Comment by jaskiratsingh2000** [comment]
Sure, go ahead

**Review by ayushnagar123 — APPROVED** [review]
Please make the required changes as suggested  by saurabh rest looks good to me

**Comment by Rupeshiya** [comment]
@ayushnagar123 @jaskiratsingh2000 @ksraj123 @vaibhavdaren @AuraOfDivinity Please review again

**Review by ksraj123 — APPROVED** [review]

**Review by AuraOfDivinity — APPROVED** [review]

**Review by ayushnagar123 — APPROVED** [review]

**Comment by Rupeshiya** [comment]
@vaibhavdaren Please review

**Pull request merged**